activerecord/test/current_page_test.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require "bundler/inline"
+
+
+require "rack/test"
+require "action_controller/railtie"
+require "minitest/autorun"
+require "action_dispatch/journey"
+
+class BugTest < ActiveSupport::TestCase
```

Comment:
I believe that it makes sense to add a test to ensure that the [case mentioned](https://github.com/rails/rails/pull/47341#pullrequestreview-1292200782) by @skipkayhil does not result in a false positive

Reply:
Added to url_helper_test